torchgen/model.py

```diff
@@ -2056,27 +2056,30 @@ def symints_to_ints(self) -> "Arguments":
         if arguments.self_arg:
             arguments = dataclasses.replace(
                 arguments,
-                pre_self_positional=[
-                    x.symint_to_int() for x in arguments.pre_self_positional
-                ],
+                pre_self_positional=tuple(
+                    [x.symint_to_int() for x in arguments.pre_self_positional]
```

Comment:
Why do you need the temporary list?

Reply:
Um, I've seen this pattern in quite a few places: https://github.com/pytorch/pytorch/blob/master/torch/autograd/functional.py#L671 

What should I do instead, `tuple((x for x in range(10)))` ?

Reply:
Just `tuple(x for x in range(10))` no need for intermediary objects.

Reply:
sounds gut, I'll make a change